litmus-portal/frontend/okteto.yml

```diff
@@ -0,0 +1,20 @@
+name: litmusportal-frontend
```

Comment:
Is it necessary to put `okteto.yml` inside frontend directory

Reply:
yes, this is equivalent to `Dockerfile` for dev container.